integrations/deploy.yaml

```diff
@@ -255,8 +255,10 @@
             -v /var/log:/host/var/log:ro \
             -v /var/run/docker.sock:/var/run/docker.sock:ro \
             --restart unless-stopped \
-            --cap-add SYS_PTRACE \
+            --cap-add DAC_READ_SEARCH \
+            --cap-add NET_RAW \
             --cap-add SYS_ADMIN \
+            --cap-add SYS_PTRACE \
```

Comment:
I'm not sure all this is necessary since plugins run with the setuid bit. I tested go.d/ping and it works without `NET_RAW`. So `NET_RAW` is not needed.